packages/docs/docs/careplans/loinc.md

```diff
@@ -0,0 +1,373 @@
+---
+keywords:
+  - loinc
+  - diagnostics
+  - lab
+  - lab codes
+  - diagnostic codes
+  - LIS/LIMS
+tags:
+  - lab
+  - diagnostics
+  - loinc
+  - care plans
+  - LIS
+  - LIMS
+---
+
+# LOINC Codes
+
+LOINC (Logical Observation Identifiers Names and Codes) was established with the intention of defining a universal standard for identifying clinical data in electronic reports. The overall scope of LOINC is anything you can **test**, **measure**, or **observe** about a patient.
+
+This guide aims to provide a basic understanding of LOINC codes and how they are applied in practice.
+
+Adopting your clinical data to LOINC codes help:
```

Comment:
Minor phrasing note, but this should probably be either: "Adapting your clinical data to LOINC" or "Adopting LOINC codes for your clinical data"